EIPS/static_call.md

```diff
@@ -0,0 +1,48 @@
+## Preamble
+
+    EIP: 214
+    Title: New opcode STATICCALL
+    Author: Vitalik Buterin <vitalik@ethereum.org>, Christian Reitwiessner <chris@ethereum.org>;
+    Type: Standard Track
+    Category: Core
+    Status: Draft
+    Created: 2017-02-13
+
+## Simple Summary
+
+To increase smart contract security, this proposal adds a new opcode that can be used to call another contract (or itself) while disallowing any modifications to the state during the call (and its subcalls, if present).
+
+## Abstract
+
+This proposal adds a new opcode that can be used to call another contract (or itself) while disallowing any modifications to the state during the call (and its subcalls, if present). Any opcode that attempts to perform such a modification (see below for details) will result in an exception instead of performing the modification.
+
+## Motivation
+
+Currently, there is no restriction about what a called contract can do, as long as the computation can be performed with the amount of gas provided. This poses certain difficulties about smart contract engineers: After a regular call, unless you know the called contract, you cannot make any assumptions about the state of the contracts. Furthermore, because you cannot know the order of transactions before they are confirmed by miners, not even an outside observer can be sure about that in all cases.
+
+This EIP adds a way to call other contracts and restrict what they can do in the simplest way. It can be safely assumed that the state of all contracts is the same before and after a static call.
+
+## Specification
+
+Introduce a new `STATIC` flag to the virtual machine. This flag is set to `false` initially. Its value is always copied to sub-calls with an exception for the new opcode below.
+
+Opcode: `0xfa`.
```

Comment:
I think we will be better off choosing random opcodes.  If we try to avoid gaps, we need to sort EIPs in a linear ordering, and more communication is needed.  On the other hand, if we allow gaps, no communication is necessary unless there is a collision (or, when one party notices a collision, they can simply change the opcode, so maybe no communicaiton is necessary).